app/assets/javascripts/admin.js.coffee

```diff
@@ -1,33 +0,0 @@
-#= require_self
```

Comment:
please put this back.. this will be used next time we need a page-specific javascript